release-notes.md

```diff
@@ -1,3 +1,17 @@
+# Ursa-21.1.0
+## Enable V2_V7 Redirect ##
```

Comment:
write it in a meanningfull way: 
"Enable player calls redirection from V2 to V7"